tests/melt_material_2.prm

```diff
@@ -0,0 +1,170 @@
+# A testcase that demonstrates melt transport with a compactible matrix,
```

Comment:
I don't really know... 
Google gave me 110,00 results for 'compactable media', and 85,000 for 'compactible media'... I've definitely seen it in use in melt migration related papers as compactible matrix.